zebrad/tests/acceptance.rs

```diff
@@ -2860,3 +2860,100 @@ fn scan_task_starts() -> Result<()> {
 
     Ok(())
 }
+
+/// Test that the scanner can continue scanning where it was left when zebrad restarts.
+///
+/// Needs a cache state close to the tip. A possible way to run it locally is:
+///
+/// export ZEBRA_CACHED_STATE_DIR="/path/to/zebra/state"
+/// cargo test scan_start_where_left --features="shielded-scan" -- --ignored --nocapture
+///
+/// The test will run zebrad with a key to scan, scan the first few blocks after sapling and then stops.
+/// Then it will restart zebrad and check that it resumes scanning where it was left.
+///
+/// Note: This test will remove all the contents you may have in the ZEBRA_CACHED_STATE_DIR/private-scan directory
+/// so it can start with an empty scanning state.
+#[ignore]
+#[test]
+#[cfg(feature = "shielded-scan")]
+fn scan_start_where_left() -> Result<()> {
```

Comment:
I toke a few shortcuts doing the test instead of adding code in `launch`, `test_type`, etc.

Reply:
That's fine in tests 🙂